src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -192,6 +207,81 @@ protected static GenotypesContext fixGenotypes(final GenotypesContext originals,
     }
 
     /**
+     * method to swap the reference and alt alleles of a bi-allelic, SNP
+     *
+     * @param vc
+     * @return
+     */
+    public static VariantContext swapRefAlt(final VariantContext vc, final Collection<String> annotationsToReverse, final Collection<String> annotationsToDrop) {
+        //TODO: assert vc is a biallelic snp
+
+        final VariantContextBuilder swappedBuilder = new VariantContextBuilder(vc);
+
+        swappedBuilder.attribute(SWAPPED_ALLELES, true);
+
+        // You need to get the baseString in order to forget who's reference and who's variant
+        swappedBuilder.alleles(Arrays.asList(vc.getAlleles().get(1).getBaseString(), vc.getAlleles().get(0).getBaseString()));
+
+
+        final Map<Allele, Allele> alleleMap = new HashMap<>();
+        for (int idx = 0; idx < vc.getAlleles().size(); idx++) {
+            // The index of the allele changed from 1 to 0 and vice versa, but the bases remain the same.
+            alleleMap.put(vc.getAlleles().get(idx), swappedBuilder.getAlleles().get(1 - idx));
```

Comment:
IMO a loop here is less clear than writing out the two `put()` calls you'd otherwise have.